### ISSUE TITLE
Validate TS doesn't match map TS

### ISSUE THREAD

**Issue by kbailey-noaa** [asascience-open/IOOS-EDS#200]
@kknee @Bobfrat 
Trying to compare CBIBS salinity vs CBOFS OFS salinity.  Time-series comparison in the map doesn't match the time-series in the validate tool.  Per the map, the time-series should intersect at some point but they never do in the validate tool.  Tried with York Spit salinity vs CBOFS as well - same issue.

![Gooses Rf Salinity map TS](https://user-images.githubusercontent.com/11823385/66328999-ac7fc200-e8fb-11e9-8770-15eedbc3a9d5.PNG)

![Gooses Rf Salinity validate](https://user-images.githubusercontent.com/11823385/66329009-b1dd0c80-e8fb-11e9-9e3c-176c3243fdce.PNG)

**Comment by kbailey-noaa** [comment]
@kknee @Bobfrat 
I don't think the CBOFS data in the map are plotting accurately.  The CBIBS Gooses Reef salinity value at 10/6 21:00 is 15.2, and CBOFS at that date/time is 15.68. Yet the Gooses Reef data at that date/time are plotting higher than the CBOFS data.

![CBIBS salinity value](https://user-images.githubusercontent.com/11823385/66332405-a4775080-e902-11e9-9392-50625d62d8e5.png)

![CBOFS salinity value](https://user-images.githubusercontent.com/11823385/66332408-a6411400-e902-11e9-8d3b-6090ce482917.png)

**Comment by Bobfrat** [comment]
is there a second axis on the left side of the plot? This is probably a units thing? In the map page page if multiple parameters are on it creates a new axis for each unique set of units. Parameters with shared units can share an axis. I suspect one has units psu and the other is unitless?

**Comment by kbailey-noaa** [comment]
@Bobfrat Yes, that's it.  Shouldn't the units be the same?  I thought they'd both be PSU (or both unitless...whatever the standard is these days).  Threw me for a loop when the axes changed between the map and validate tool, changing the way the time-series curves looked together.

**Comment by Bobfrat** [comment]
Haha yeah the units seem to be a debated topic. I've seen PSU, PPT, g/kg . I'm not sure what's right anymore. But the issue here is a units discrepancy so we'll have to address the units.

**Comment by kbailey-noaa** [comment]
Roger that. I changed the title to reflect the units question, and assigning this to a different milestone.

**Comment by kknee** [comment]
@kbailey-noaa the latest from Mid-A stakeholders has been unitless salinity.  Are you okay with us moving to that?

**Comment by kbailey-noaa** [comment]
@kknee Yes, definitely. I think I posed the question to some listserv or modeling group at one point a year or two ago and they insisted salinity should be unitless, so I'm glad to hear the feedback is consistent!
